sdk/core/amqp-common/README.md

```diff
@@ -61,7 +55,7 @@ You can run the samples by cloning the repo or copy pasting the below sample in
 ```
 
 The samples below are generic for EventHubs and Servicebus. You can find EventHub specific samples,
-in the [samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/core/amqp/samples) directory.
+in the [samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/core/amqp-common/samples) directory.
```

Comment:
@ramya0820 
Can you update that link to point to the release tag instead so that it will be permanent? https://github.com/Azure/azure-sdk-for-js/tree/%40azure/amqp-common_1.0.0-preview.4/sdk/core/amqp-common/samples